src/sage/rings/polynomial/polynomial_ring.py

```diff
@@ -1386,8 +1390,8 @@ def random_element(self, degree=(-1,2), *args, **kwds):
             sage: R.random_element(6).degree()
             6
 
-        If a tuple of two integers is given for the ``degree`` argument, a degree
-        is first uniformly chosen, then a polynomial of that degree is given::
+        If a tuple of two integers is given for the ``degree`` argument, a polynomial is chosen
+        uniformly among all polynomials with degree between them::
```

Comment:
```suggestion
        among all polynomials with degree between them::
```
I am dubious about this uniform distribution claim since it depends highly on the sampling of the coefficient ring (which doesn't have to be uniform).

Reply:
I think the information is still important to note for cases like $$\mathbb{F}_{103}[x]$$ which is uniform. I now phrase it as

```
If a tuple of two integers is given for the ``degree`` argument, a polynomial is chosen
among all polynomials with degree between them. If the base ring is uniform, so is this
method::
```

is that ok? Not sure if the English is really correct.

Reply:
I think the last sentence needs to be something like "If the base ring is sampled uniformly, then this is also sampled uniformly::"

I am wondering if we should include a test about the uniformness. It's hard to test it reliably is random, but perhaps we should sample over something like 10000 polynomials of degree 3 over $F_3$ and check the distribution for the first digit? I don't think this is enough for 2 digits in general. It might not be worth it to spend a lot of time on this.

Reply:
I agree that we should have a test for uniformness, but I don't know how to approach this correctly. I tried a chi-squared test with say $p = 0.05$, but it often fails, c.f. ["Too Big to Fail: Large Samples and the p-Value Problem"](https://www.researchgate.net/publication/270504262_Too_Big_to_Fail_Large_Samples_and_the_p-Value_Problem).

Reply:
Yea, I couldn't think of a good way to reasonably guarantee this. Let's just leave it.

However, it doesn't make sense to say "the base ring is uniform". You need to discuss about the sampling. My previous suggestion can also be improved by "If we can sample uniformly from the base ring, then ..."

Reply:
This still needs to be address. It does not make sense to say the base ring is uniform.

Reply:
Oh god, I completely missed this comment.